snuba_sdk/legacy.py

```diff
@@ -242,8 +244,9 @@ def json_to_snql(body: Mapping[str, Any], entity: str) -> Query:
                     Condition(Column(time_column), op, parse_datetime(date_val))
                 )
 
-    if body.get("project"):
-        proj_cond = parse_extension_condition("project_id", body["project"], True)
+    project_id = body.get("project") or body.get("project_id")
+    if project_id:
+        proj_cond = parse_extension_condition("project_id", project_id, True)
```

Comment:
Why this change ?

Reply:
Subscriptions use `project_id` as a key instead of `project`.